docs/docs/40-operator-guide/40-security/40-managing-credentials.md

```diff
@@ -255,58 +255,79 @@ obtain a token that is valid for 12 hours and cached for 10.
 
 ### Google Artifact Registry
 
-Kargo can be configured to authenticate to Google Artifact Registry repositories
-using Workload Identity Federation.
+Kargo can be configured to authenticate to
+[Google Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview)
+(GAR) repositories using
+[Workload Identity Federation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity))
+(WIF).
+
+#### Cluster Configuration
 
 If Kargo locates no `Secret` resources matching a repository URL, and if Kargo
-is deployed within a GKE cluster, it will attempt to use
-[Workload Identity Federation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
-to authenticate, but this relies upon some external setup. Leveraging this
-option eliminates the need to store credentials in a `Secret` resource.
-
-First, follow
-[these directions](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_on_cluster)
-to provision a new GKE cluster with Workload Identity Federation enabled or
-[these directions](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable-existing-cluster)
-to enable Workload Identity Federation on an existing GKE cluster.
-
-At this point, the `kargo-controller` `ServiceAccount` within the namespace in
-which Kargo is (or will be) installed will be associated with an _IAM principal
-identifier_, which takes the following form:
-
-```plaintext
-principal://iam.googleapis.com/projects/<gcp project number>/locations/global/workloadIdentityPools/<gcp project name>.svc.id.goog/subject/ns/<kargo namespace>/sa/kargo-controller
-```
+is deployed within a
+[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview)
+(GKE) cluster, it will attempt to use WIF to authenticate. Leveraging this
+option eliminates the need to store credentials in a `Secret` resource. However, this requires WIF to be enabled on the cluster. You can [enable this when the cluster is created](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_on_cluster) or you can [enable it on an existing cluster that doesn't already have it](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable-existing-cluster). 
```

Comment:
```suggestion
--------------------------------------------------------------------------------
(GKE) cluster with WIF enabled, it will attempt to use it to authenticate.
Leveraging this option eliminates the need to store credentials in a `Secret`
resource. WIF can be enabled when creating a
[new cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_on_cluster)
or can be added to an
[existing cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable-existing-cluster).
```